components/resource/credits.vue

```diff
@@ -1,33 +1,43 @@
 <template>
-  <div v-if="producers.length">
-    <p class="fr-text--bold fr-mb-0 fr-text--sm">Crédits de la ressource</p>
-    <p
-      v-for="producer in producers"
-      :key="producer.name"
-      class="fr-text--sm fr-m-0"
-    >
-      <template v-if="producer.href">
-        <template v-if="producer.isExternalLink">
-          <a
-            :href="producer.href"
-            class="fr-text-label--blue-france no-underline underlined-on-hover"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            {{ producer.name }}
-          </a>
+  <div>
+    <h2 class="fr-text--bold fr-mb-2w fr-text--md">Crédits de la ressource</h2>
+    <div v-if="producers.length">
+      <p
+        v-for="producer in producers"
```

Comment:
Quand il y a un v-for j'aime bien créer un composant intermédiaire pour simplifier la relecture
Après le composant crédit est pas immense donc ca me gène pas